### PR TITLE
Refactor build sh py3.5

### PR DESCRIPTION
**Pull request by xmnlab**
Changes from #107

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -1,19 +1,27 @@
...
+    - omit-local-site-packages.patch  # [osx]
+    - osx64-dist.patch                # [osx and x86_64]
+    - win-find_exe.patch              # [win]
+    - win-library_bin.patch           # [win]
+    # Adjust FD_SETSIZE to allow select() to wait on more connections
+    # simultaneously.  Desirable for serious use of Tornado on Windows.
+    # See also:
+    # - https://bugs.python.org/issue28708
+    # - https://github.com/zeromq/libzmq/issues/1165
+    - fd_setsize.patch                # [win]
```
Comment (ocefpaf) [inline recipe/meta.yaml]: It seems that your PR is missing this file.

@jjhelmus lost track of this. Should @xmnlab remove it or commit the patch here too?

**Comment by ocefpaf** [comment]
We need to address https://github.com/conda-forge/python-feedstock/pull/109#discussion_r96114742 to get a passing Appveyor here.

File: recipe/meta.yaml (hunk 1 above)
Comment (jjhelmus) [inline recipe/meta.yaml]: I would commit the patch.  It is included in Python 3.6.0 and should be included in 3.5.x.

**Comment by jjhelmus** [comment]
I added the `fd_setsize.path` file from the `master` branch in 0ca4de1.  Lets see if that fixes the Windows build.

**Comment by xmnlab** [comment]
All checks have passed
4 successful checks

:)

**Comment by ocefpaf** [comment]
Thanks @xmnlab for the PR and @jjhelmus for the patch fix!

**Pull request merged**